provisioning/miniprov/hare_mp/main.py

```diff
@@ -551,13 +548,15 @@ def init(args):
         # Starting consul and hax
         consul_starter = _start_consul(utils, stop_event,
                                        config_dir, log_dir, url)
-        hax_starter = _start_hax(utils, stop_event, config_dir, log_dir)
         hostname = utils.get_local_hostname()
-        # Cleanup old mkfs state
-        cleanup_mkfs_state(utils, cns_utils)
-        start_mkfs_parallel(hostname, config_dir)
-        # Update mkfs state
-        set_mkfs_done_for(hostname, cns_utils)
+        if is_mkfs_required(url):
+            hax_starter = _start_hax(utils, stop_event, config_dir, log_dir)
```

Comment:
we still need to start hax in server pod, why is it avoided?

Reply:
@mssawant In 'init' stage?